### PR TITLE
Add documentation for `hx --health`

### PR DESCRIPTION
**Pull request by RobWalt**
This is a draft for the solution of #1822 

I took all ideas from the issue and:

- added a troubleshooting page to the book with section about healthcheck
- added a remark at end of installation page regarding `hx --health` pointing
to troubleshooting page mentioned above

Would be cool if a person with better english skills than mine could check the few lines I wrote.

### CODE REVIEW

**Comment by the-mikedavis** [comment]
I think we should link to the troubleshooting wiki entry: https://github.com/helix-editor/helix/wiki/Troubleshooting and probably the FAQ page as well instead of adding a new troubleshooting doc. Troubleshooting info tends to need to be updated more often than docs otherwise, so if people are looking at the release documentation rather than the master docs they may miss a pitfall.

**Comment by RobWalt** [comment]
Yeah totally makes sense to me.

So should we move the documentation of the `--health` argument there aswell under a new section?

**Comment by the-mikedavis** [comment]
Ah yep that's a good idea 👍

**Comment by RobWalt** [comment]
Alright, awesome. I will take care about it and propose a draft for the changes for the wiki. Thanks for the feedback.

File: book/src/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- the log file 
+- the runtime directory
+
+### Language Configuration Section 
+
+The second part of the healthcheck output consists of a table which shows the current status of each language supported by Helix. There are several features which can be configured for each language. The list of the features are:
+
+- LSP (Language Server Protocol)
+- DAP (Debug Adapter Protocol)
+- Highlight (???)
```
Comment (archseer) [inline book/src/troubleshooting.md:26]: Syntax highlighting

**Comment by RobWalt** [comment]
I propose adding an extra page "Healthcheck" in the Wiki with the following content:

> # Healthcheck
> 
> To make sure your Helix installation is set up the way you intented it to be, make sure to run 
> ```
> hx --health
> ``` 
> The output is split into two sections:
> 
> ## Helix configuration section 
> 
> The first part of the healthcheck output consists of the locations of all the important files currently used by Helix. These include:
> 
> - the config file 
> - the language file 
> - the log file 
> - the runtime directory
> 
> ## Language configuration section 
> 
> The second part of the healthcheck output consists of a table which shows the current status of each language supported by Helix. There are several features which can be configured for each language. The list of the features are:
> 
> - LSP (Language Server Protocol)
> - DAP (Debug Adapter Protocol)
> - Highlight (Syntax highlighting)
> - TextObject (jump to functions, classes, etc. [details here](./guides/textobject.md))
> - Indent ([details here](./guides/indent.md))
> 
> The color of the entry in the table indicates the current state of the feature:
> 
> - <b style="color:red">red</b>: The feature cannot be found on the system
> - <b style="color:yellow">yellow</b>: The feature isn't configured currently
> - <b style="color:green">green</b>: The feature was found on the system
> 
> Additionally you can run:
> ```
> hx --health [LANG]
> ```
> to get a more detailed overview for the configuration status of a specific language. Additionally to the features listed above, the output contains:
> 
> - the location of the LSP binary (if available)
> - the location of the DAP binary (if available)

Then we can put a link to this page in the book aswell as at the start of the troubleshooting wiki page. I think the extra page in the wiki for healthcare will increase its visibility and help people fix the "obvious" errors more quickly.

If you give me the thumbs up, I'll apply the changes to the wiki.

**Comment by the-mikedavis** [comment]
That looks great! Please go ahead and add it 👍 

The only thing I think needs changing is that the links to the guides should go to the docs website rather than the relative paths that would have been used in the book (e.g.  https://docs.helix-editor.com/master/guides/indent.html for the indent guide)

**Comment by RobWalt** [comment]
Oh I missed that, thanks for pointing out!

Wiki page is added ✅

**Review by dead10ck — APPROVED** [review]
Looks good to me, thanks!

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks!

**Pull request merged**